modules/app-configuration/configuration-stores/key-values/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "App Configuration",
```

Comment:
Should the README.md be updated to "App Configuration Store Key Values". 
"This module deploys an App Configuration Store Key Value."